src/emonesp.h

```diff
@@ -5,15 +5,7 @@
 // General support code used by all modules
 // -------------------------------------------------------------------
 
-// Uncomment to use hardware UART 1 for debug else use UART 0
-// #define DEBUG_SERIAL1
```

Comment:
Ok, good, this fixes that regression I mentioned elsewhere but I'd still liketo know what happened therek and what else may have been (inadvertently?) changed ..